test/persona_client_tests.js

```diff
@@ -145,18 +145,6 @@ describe("Persona Client Test Suite - Constructor & Token Tests", function() {
                     });
                 });
             });
-            it("should not return a token",function(done) {
-                var personaClient = persona.createClient("test-suite",personaClientConfig);
-
-                personaClient.obtainToken({id: "primate", secret: "wrong_password"},function(err, data) {

```

Comment:
Didn't the cache key used to contain the secret? How do we cover regressions here.